pages/api/zapbye.ts

```diff
@@ -0,0 +1,98 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import jsforce from 'jsforce';
+
+const CONSTANTS = {
+      SALESFORCE_EMAIL: process.env.SALESFORCE_EMAIL,
+      SALESFORCE_PASSWORD: process.env.SALESFORCE_PASSWORD,
+      SALESFORCE_SECURITY_TOKEN: process.env.SALESFORCE_SECURITY_TOKEN,
+      LEMLIST_API_KEY: process.env.LEMLIST_API_KEY,
+      SALESFORCE_TASK_SUBJECT: 'Lemlist Email',
+      SALESFORCE_LEMLISTTASK_ID: '0123k0000014TCyAAM',
+      SALESFORCE_STATUS: 'Completed',
+      SALESFORCE_TASK_SUBTYPE: 'Task',
+      SALESFORCE_SOBJECT_TYPE: 'Task',
+      LEMLIST_CAMPAIGN_FETCH_COOLDOWN_TIME_IN_MS: 30000,
+};
+
+let salesforceConnection;
+let lastCampaignCheckedTimestamp: number;
+let cachedCampaignsArray: Array<LemlistCampaign> = [];
```

Comment:
I like this syntax, you can also 
```ts
let cachedCampaignsArray: LemlistCampaign[] = [];
```

I don't have a preference either way

Reply:
I prefer To have al encapsulating types wrap the encapsulated types since I find it more intuitive when reading, but if we ever want to standarize I don't have super strong opposition over using the standard array syntax.